src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -25,9 +25,19 @@
   become: yes
   user: name="{{ nginx_user }}" createhome=no password=no state=present groups="{{ cchq_user }}"
 
+- stat:
+    path: "{{ code_home }}"
+  register: code_home_folder
```

Comment:
Not super happy with this solution because there's always a possibility (though unlikely) that we could move staticfiles out of code_home. With that said, I don't think that will actually happen, so I like this as a simple safeguard to not creating this folder too early